README.md

```diff
@@ -1,181 +1,78 @@
-# Tracee
-Container and system tracing using eBPF
+# Tracee - Container and system tracing using eBPF
 
-**Tracee** is a lightweight, easy to use container and system tracing tool.
-After launching the tool, it will start collecting traces of newly created containers (container mode) or processes (system mode).
-The collected traces are mostly system calls performed by the processes,
-but other events, such as capabilities required to perform the actions requested by the process, are also supported.
+Tracee is a lightweight and easy to use container and system tracing tool. It allows you to observe system calls and other system events in real time. A unique feature of Tracee is that it will only trace newly created processes and containers (that was started after Tracee has started), in order to help the user focus on relevant events instead of every single thing that happens on the system (which can be overwhelming).
```

Comment:
```suggestion
Tracee is a lightweight and easy to use container and system tracing tool. It allows you to observe system calls and other system events in real time. A unique feature of Tracee is that it will only trace newly created processes and containers (that were started after Tracee has started), in order to help the user focus on relevant events instead of every single thing that happens on the system (which can be overwhelming).
```